src/test/java/com/wikia/webdriver/elements/fandom/components/LoadMore.java

```diff
@@ -17,7 +17,7 @@ public boolean isButtonDisplayed() {
         try {
             return loadMoreButton.isDisplayed();
         } catch (ElementNotFoundException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)